modules/valkey/valkey.go

```diff
@@ -57,7 +57,7 @@ func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustom
 	req := testcontainers.ContainerRequest{
 		Image:        img,
 		ExposedPorts: []string{"6379/tcp"},
-		WaitingFor:   wait.ForLog("* Ready to accept connections"),
+		WaitingFor:   wait.ForListeningPort("6379/tcp"),
```

Comment:
question: do you think we could keep both conditions, the log entry and the listening port?

Reply:
That would only add more work, so keeping it simple would be my preference.